packages/core/src/tracing/trace.ts

```diff
@@ -157,7 +157,7 @@ export function startInactiveSpan(options: StartSpanOptions): Span {
   // If `options.parentSpan` is defined, we want to wrap the callback in `withActiveSpan`
   const wrapper = options.scope
     ? (callback: () => Span) => withScope(options.scope, callback)
-    : customParentSpan
+    : customParentSpan !== undefined
```

Comment:
hmm I'm not sure I understand the PR description correctly but to me it looks like we're changing 2 things in this PR:

1. make `options.parentSpan` accept `null`. This sounds fine to me, given we're aligning our public API better
2. call `withActiveSpan(null,...)` in case `parentSpan` is `null`. Is this necessary? Shouldn't the behaviour be identical to simply invoking the `callback`?

Reply:
no, it is important to call `withActiveSpan` in this case - if you pass `null`, the meaning is `start this span as a root span without a parent`, while` undefined` means the default behavior (the parent is the current active span). Does that make sense?

Reply:
See https://docs.sentry.io/platforms/javascript/tracing/instrumentation/custom-instrumentation/#withactivespan

Reply:
missed this, thx for explaining!